netcontrold/tests/lib/test_util.py

```diff
@@ -19,8 +19,18 @@
 from netcontrold.lib import util
 
 _BASIC_CPU_INFO = """
-processor	: 0
+processor       :
```

Comment:
check for indentation before ":" .. it has to be consistent (when the original code is keeping it for some length).